submissions/lukmanokunade/HTML/Registration.html

```diff
@@ -2,21 +2,33 @@
 <html>
     <head>
         <title>Deuces</title>
+        <link href="style.css" rel="stylesheet">
     </head>
+
     <body>
-        <h1>Description</h1>
-        <p>If you would like to experience the best of online shopping for men, women and kids, you are<br> at the right place. Deuces is the ultimate destination for fashion and lifestyle,<br> being host to a wide array of merchandise including clothing, footwear, accessories, jewellery,<br> personal care products and more. It is time to redefine your style statement with our treasure-trove of trendy items. <br>Our online store brings you the latest in designer products straight out of fashion houses. <br>You can shop online at Deuces from the comfort of your home and get your favourites delivered right to your doorstep.</p>
+    <h1>Description</h1>
+        <p>If you would like to experience the best of online shopping for men, women and kids, you are at the right place.<br>
+            Deuces is the ultimate destination for fashion and lifestyle, being host to a wide array of merchandise including<br>
+            clothing, footwear, accessories, jewellery, personal care products and more. It is time to redefine your style statement<br>
+            with our treasure-trove of trendy items. Our online store brings you the latest in designer products straight out of fashion houses.<br>
+            You can shop online at Deuces from the comfort of your home and get your favourites delivered right to your doorstep.</p>
         <h1>Create an Account</h1>
         <form>
             <fieldset>
-            First Name <input type="First name" name="First name" id="First name">
+        <centre>
+            First Name/Given name<br>
+             <input type="First name" name="First name" id="First name" placeholder="Enter First name">
             <br><br>
-            Last name <input type="Last name" name="Last name" id="Last name"/><br><br>
-            Email Address <input type="email address" name="email address" id="email address"><br><br>
-            Phone number <input type="number" name="phone number" id="phone number"> <br><br>
-            Create password <input type="password" name="password" id="password">
-            <br><br>Confirm Password <input type="password" name="Password" id="password"> <br><br>
+            Last name/Family name <br> <input type="Last name" name="Last name" id="Last name" placeholder="Enter Family name"/>
```

Comment:
Please format this section of your markup. You can refer to the styleguide [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md#html) to see some tips on how to.